### PR TITLE
[gtk3] Add new/old port gtk3

### PR DESCRIPTION
**Pull request by talregev**
[gtk3] Add new/old port gtk3.

Please help me to make this port useable again. Thank you.

### CODE REVIEW

**Comment by JackBoosY** [comment]
Seems we already have a PR to add gtk3: https://github.com/microsoft/vcpkg/pull/22201

**Comment by talregev** [comment]
> Seems we already have a PR to add gtk3: #22201

Yes, but it have a lot of troubles, and it take a lot of time to merge. This old one already approve to master. I just try to revive it.
Also @dg0yt Tell me if I want to talk with "we", I should do it first.
So, @dg0yt  you are welcome to help me as well.

**Comment by talregev** [comment]
It not find the:
`find_library(LIBINTL_LIBRARY`
@dg0yt @JackBoosY Can you help?

**Comment by talregev** [comment]
I did as much as I can. I am not sure why the old port don't work out of the box.
Now there many linkage and compilation errors. At least it find all the libraries.

**Comment by JackBoosY** [comment]
Will handle this PR later.

File: ports/gtk3/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    PkgConfig::PANGO
+    PkgConfig::GDK_PIXBUF
+    PkgConfig::ATK
+)
+
+if (WIN32)
+    set(GTK_REQUIRED_LIBRARIES
+        ${GTK_REQUIRED_LIBRARIES}
+        PkgConfig::PANGO_WIN32
+    )
```
Comment (JackBoosY) [inline ports/gtk3/CMakeLists.txt:57]:
```suggestion
    list(APPEND GTK_REQUIRED_LIBRARIES
        ${GTK_REQUIRED_LIBRARIES}
        PkgConfig::PANGO_WIN32
    )
```

File: ports/gtk3/CMakeLists.txt [hunk 2]
```diff
@@ -51,7 +51,7 @@ set(GTK_REQUIRED_LIBRARIES
 )
 
 if (WIN32)
-    set(GTK_REQUIRED_LIBRARIES
+    list(APPEND GTK_REQUIRED_LIBRARIES
```
Comment (talregev) [inline ports/gtk3/CMakeLists.txt:54]: Now we have the list twice?

File: ports/gtk3/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+    PkgConfig::CAIRO
+    PkgConfig::CAIRO_GOBJECT
+    PkgConfig::PANGO
+    PkgConfig::GDK_PIXBUF
+    PkgConfig::ATK
+)
+
+if (WIN32)
+    list(APPEND GTK_REQUIRED_LIBRARIES
+        ${GTK_REQUIRED_LIBRARIES}
```
Comment (talregev) [inline ports/gtk3/CMakeLists.txt:55]: Now we have the list twice?

Comment (JackBoosY) [inline ports/gtk3/CMakeLists.txt:55]:
```suggestion
```

Comment (JackBoosY) [reply ports/gtk3/CMakeLists.txt]: Forgot to remove this line.

**Comment by JackBoosY** [comment]
The section about adding source files in CMakeLists.txt must be refactored to fully accommodate all systems.

**Comment by talregev** [comment]
> The section about adding source files in CMakeLists.txt must be refactored to fully accommodate all systems.

Can you do that?

**Pull request closed without merging**